Global.h

```diff
@@ -698,3 +698,16 @@ namespace vstd
 }
 using vstd::operator-=;
 using vstd::make_unique;
+
+#if defined VCMI_ANDROID && defined NO_STD_TOSTRING

```

Comment:
Oh I`ve forgotten about -std=gnu++11. I use that from the very beginning for MinGW.